Makefile

```
@@ -19,7 +19,7 @@ VENV ?= .venv
 # --- PHONY TARGETS ---
 # .PHONY ensures that make will run the command even if a file with the same
 # name as the target exists.
-.PHONY: all setup start debug devcontainer format lint fix clean help
+.PHONY: all setup start debug devcontainer test-server test-mqtt-printer test-mqtt-broker format lint fix clean help
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)